src/__tests__/patients/related-persons/AddRelatedPersonModal.test.tsx

```diff
@@ -33,25 +26,23 @@ describe('Add Related Person Modal', () => {
     dateOfBirth: new Date().toISOString(),
   } as Patient
 
-  let store: MockStore
+  let wrapper: ReactWrapper
 
-  describe('layout', () => {
-    let wrapper: ReactWrapper
-
-    store = mockStore({
-      patient: { patient },
-    } as any)
-    beforeEach(() => {
-      jest.spyOn(PatientRepository, 'find').mockResolvedValue(patient)
-      jest.spyOn(PatientRepository, 'saveOrUpdate').mockResolvedValue(patient)
-
-      wrapper = mount(
-        <Provider store={store}>
-          <AddRelatedPersonModal show onCloseButtonClick={jest.fn()} toggle={jest.fn()} />
-        </Provider>,
-      )
-    })
+  beforeEach(() => {
+    jest.spyOn(PatientRepository, 'find').mockResolvedValue(patient)
+    jest.spyOn(PatientRepository, 'saveOrUpdate').mockResolvedValue(patient)
 
+    wrapper = mount(
+      <AddRelatedPersonModal
+        show
+        patientId={patient.id}
+        onCloseButtonClick={jest.fn()}
+        toggle={jest.fn()}
+      />,
+    )
+  })
```

Comment:
would prefer this before each to be moved to a `setup` function similar to what we do in other tests to make sure a new wrapper is setup each time.